code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -337,18 +337,27 @@ CONTENTS:
 
 	add_attack_logs(user, L, "Stunned with [src]")
 
-/obj/item/abductor_baton/proc/SleepAttack(mob/living/L,mob/living/user)
-	if(L.IsStunned() || L.IsSleeping())
-		L.visible_message("<span class='danger'>[user] has induced sleep in [L] with [src]!</span>", \
+/obj/item/abductor_baton/proc/SleepAttack(mob/living/L, mob/living/user)
+	var/mob/living/carbon/C = L
+	if(!iscarbon(L))
+		return
+	if(C.IsSleeping())
+		C.visible_message("<span class='notice'>[C] is already asleep!</span>")
+		return
```

Comment:
It used to work while they were sleeping, please
- revert to old functionality
or
- include in changelog and inform design/balance team of changes

Reply:
I have no idea why I thought this was an exclude check. guh. Fixed in latest.